Templates/ScriptCanvasNode/Template/${Name}_files.cmake

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
```

Comment:
This is and the .cmake files is missing the `${BEGIN_LICENSE}` and `${END_LICENSE}` sections around the copyright